pkg/controller/cassandradatacenter/controller/constants.go

```diff
@@ -0,0 +1,10 @@
+package controller
+
+// Constants here are used in more than one go file.
+
+const (
+	CassandraInternodePort  = 7000
+	CassandraCqlPort        = 9042
```

Comment:
Minor nit: both CQL and JMX are acronyms, yet one is defined as `Cql` the other `JMX`. It's either `CQL`/`JMX` or `Cql`/`Jmx`.

Reply:
sure, will fix